### DISCUSSION TITLE
Lenovo 13X G2 - problem with speakers (cirrus cs35l41) - how to add my device to patch?

### DISCUSSION THREAD

**Discussion by PiCkLeS76** [xanmod/kernel#428] [Q&A]
Hi,

everywhere I look, hundred of posts, about my problem that my speakers doesnt work on my model of laptop, there are many laptops with this problem that are added via patches like this below to get them to work, this one in xanmod latest release of x86v3:

https://github.com/xanmod/linux/commit/6e9f02e69d9dd0b6486251f8852c79e3a57689f9

Where do I send my requests to either xanmod or the official kernel team or cirrus etc to get my laptop model also supported?

Sorry for bad english and the highest level of laymanship..

Niclas

**Comment by xanmod** [comment]
https://github.com/xanmod/linux/commit/66a36d99f9835bab32e4b5461ad36f4d6d2ccac7
https://github.com/xanmod/linux/commit/bf2dc2c9c285616a9e648919f571fa8549749546
